### DISCUSSION TITLE
Background dim comes in too late on beatmaps that have a skip button

### DISCUSSION THREAD

**Discussion by un2wo** [ppy/osu#13145] [Ideas]
On beatmaps where the intro is long enough for a skip button to appear, the background starts at a brighter dim level and darkens down to the user's preferred dim level when the gameplay starts.

This is fine and all (it seems intentional), but the dimming happens a few moments after hitting the skip button, so it feels weirdly delayed, like the game is lagging behind a bit--even though it isn't.

https://streamable.com/giexhv

**Comment by peppy** [comment]
I'm pretty sure this is just the skip button taking you to a point well before when gameplay starts (lead-in) and the fade being actuated much closer to gameplay start. It's the same as if you were to not hit the skip button.

What are you hoping for here – having skip take you closer or having the fade start earlier, even when not skipped?

**Comment by un2wo** [comment]
Having the fade start earlier. I'm aware that the fade starts at the same point in time when not skipping.

Background dim is applied at the start on maps that do not have a skip button, which is why the dim feels "unresponsive" in this situation.

edit: actually-- my issue is that the fade is sudden, but it kinda starts in the middle of things? like there's blank space between the fade and the gameplay start, unfitting of the duration of the fade animation itself.
making it start even later or having the fade be slower would be better, since bringing it closer to the skip button might still feel weird for people that don't skip.
